app/src/androidTest/java/ch/epfl/sdp/MockUserDataController.java

```diff
@@ -19,4 +19,9 @@ public void syncCloudFirebaseToRoom(LeaderoardViewModel leaderoardViewModel) {
     public void storeUser(Player player) {
         userData.add(player);
     }
+
+    @Override
+    public void joinLobby(Player player) {
+
+    }
```

Comment:
For which reason do you need a mock userDataController?

Reply:
Unless I am wrong, I didn't see you using it anywhere. I'm not saying it's wrong or anything, I just want to understand why it's here

Reply:
Because we can't test the firebase directly. Its a external ressources, thats why we are using the mock.

Reply:
At this point, the Mock one is not used, but later it will be used. So we added it here. Basically the MockUserDataController will have the same interface as FirestoreUserData,  performing as the firebase locally for test purposes.